addons/test_mail/tests/test_invite.py

```diff
@@ -16,14 +16,21 @@ def test_invite_email(self):
             'name': 'Valid Lelitre',
             'email': 'valid.lelitre@agrolait.com'})
 
-        mail_invite = self.env['mail.wizard.invite'].with_context({
-            'default_res_model': 'mail.test.simple',
-            'default_res_id': test_record.id
-        }).with_user(self.user_employee).create({
-            'partner_ids': [(4, test_partner.id), (4, self.user_admin.partner_id.id)],
-            'notify': True})
+        mail_invite = (
```

Comment:
Seems old writing took less place, could probably keep it (not sure why so much changes ?)

Reply:
it's the python formatting that we are trying to follow. but if you prefer the old style i can revert the change

Reply:
Yup because it is not something we want actually. Don't enforce guidelines that are generate noise, see https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html for official guidelines.